pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/TransactionsBase.java

```diff
@@ -501,7 +501,11 @@ protected void internalGetCoordinatorInternalStats(AsyncResponse asyncResponse,
                         "This Broker is not configured with transactionCoordinatorEnabled=true."));
             }
         } catch (Exception e) {
-            asyncResponse.resume(new RestException(e.getCause()));
+            if (e instanceof ExecutionException) {
```

Comment:
Please check the method `resumeAsyncResponseExceptionally`, seems that it already handles the `ExecutionException` case, we don't need to check it here.